app/assets/javascripts/comfortable_mexican_sofa/admin/modules/CategoryExpand.js

```diff
@@ -0,0 +1,72 @@
+define(['jquery','DoughBaseComponent'], function ($, DoughBaseComponent) {
+'use strict';
+
+  var defaultConfig = {
+    categoriesCharacterCount: 20
+  };
+  var CategoryExpandProto;
+
+  function CategoryExpand($el, config) {
+    CategoryExpand.baseConstructor.call(this, $el, config, defaultConfig);
+  }
+
+  DoughBaseComponent.extend(CategoryExpand);
+
+  CategoryExpand.componentName = 'CategoryExpand';
+
+  CategoryExpandProto = CategoryExpand.prototype;
+
+  CategoryExpandProto.init = function(initialised) {
+
+    this.addExpanders();
+
+    this.addListeners = function() {
```

Comment:
Sorry this isn't quite right. I meant to make it a new method like `CategoryExpandProto.addExpanders = function() {` is, and then call it from here with `this.addListeners()`. Does that make sense?